api/src/main/java/au/org/aodn/nrmn/restapi/service/upload/SurveyContentsHandler.java

```diff
@@ -131,13 +131,20 @@ public void cell(String cellReference, String formattedValue, XSSFComment commen
         }
     }
 
-    private String truncateDecimalString(String decimalString) {
-        int idx = decimalString.indexOf('.');
-        int endIdx = idx + 6;
-        if (idx >= 0 && endIdx < decimalString.length())
-            return decimalString.substring(0, endIdx);
-        else
+
+    private static String roundDecimalString(String decimalString) {
```

Comment:
Sorry forgot to comment this:

is this shorter with common-math (already included in pom) -> Precision.round(numnber, 5);

By default, it is using the same HALF_UP rounding method as our helper method; therefore, the results should be the same.

then format with 
DecimalFormat df = new DecimalFormat("###.###");
System.out.println(df.format(PI));

https://www.baeldung.com/java-round-decimal-number

Reply:
One thing also puzzle me is that before the change, the save() should save the incorrect value, why the auto test not picking it up?